packages/shared/ReactSharedInternals.js

```diff
@@ -10,4 +10,19 @@ import React from 'react';
 const ReactSharedInternals =
   React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED;
 
+// Add fallback for newer renderers running with older react package versions.
+// Current owner and dispatcher used to share the same ref,
+// but PR #14548 split them out to better support the react-debug-tools package.
+if (!ReactSharedInternals.hasOwnProperty('ReactCurrentDispatcher')) {
+  const {ReactCurrentOwner} = ReactSharedInternals;
+  ReactSharedInternals.ReactCurrentDispatcher = {
```

Comment:
Can't this just be `{current: null}`? Why does it need to forward to the owner object?

Reply:
Because I wasn't convinced that no 16.8+ renderer would ever have a valid use case for accessing current owner?

Reply:
e.g. react-cache uses it